userguide/jails.rst

```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The quickest and easiest way to install software inside the jail is to
-install a FreeBSD package. FreeBSD packages are pre-compiled.  They
-contains all the binaries and a list of dependencies required for the
+install a FreeBSD package. FreeBSD packages are pre-compiled. They
+contain all the binaries and a list of dependencies required for the
 software to run on a FreeBSD system.
 
-A huge amount of software has been ported to FreeBSD, currently over
-24,000 applications, and most of that software is available as a
-package. One way to find FreeBSD software is to use the search bar at
+A huge amount of software is ported to FreeBSD. Most of that
+software is available as a package. One way to find FreeBSD software is
+to use the search bar at
 `FreshPorts.org <https://www.freshports.org/>`__.
 
 After finding the name of the desired package, use the
 :command:`pkg install` command to install it. For example, to install
-the audiotag package, use this command:
-
-.. code-block:: none
-
-   pkg install audiotag
-
+the audiotag package, use the command :command:`pkg install audiotag`
 
 When prompted, type **y** to complete the installation. The
-installation messages will indicate if the package and its
+installation messages indicate if the package and
 dependencies successfully download and install.
 
 .. warning:: Some older versions of FreeBSD used package systems
```

Comment:
Without specifics, this is useless to the reader.  The package changeover was so long ago that this is not needed any more anyway.